nrf_security: cracen: make IKG seed KMU slot configurable (#17381)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

Nordic Secure Immutable Bootloader (NSIB, B0, or B0n)
-----------------------------------------------------

.. toggle::

>    Custom printing has been dropped in favor of using the logging subsystem, with output printed out to the default logging device.

Reviewer: Moving this text to under toggle will result in text not showing up... It only displays bullet points it seems :)

Author: I did that because it seems that migration guide areas are meant to be like this (most are this way). Or what do you mean exactly, is there something wrong with this particular instance? There might have been a missing newline. I fixed that.

Reviewer: My bad, it was the missing line, I tested it, but never got back to you.

Nordic Secure Immutable Bootloader (NSIB, B0, or B0n)
-----------------------------------------------------

.. toggle::

>    Custom printing has been dropped in favor of using the logging subsystem, with output printed out to the default logging device.

Reviewer: ```suggestion
   Custom printing is replaced by the logging subsystem, with output printed out to the default logging device.
```

Author: This is not something I wrote. All I did here was add the toggle...

Reviewer 2: This was reviewed and approved by Anna in https://github.com/nrfconnect/sdk-nrf/pull/17079

   The ``CONFIG_SECURE_BOOT_DEBUG`` Kconfig option has been removed.
   To disable logging in B0 or B0n, set the :kconfig:option:`CONFIG_LOG` option to ``n``.
   To send logs over RTT instead of UART, apply the following settings:

       * Enable the :kconfig:option:`CONFIG_USE_SEGGER_RTT` and :kconfig:option:`CONFIG_RTT_CONSOLE` Kconfig options.

>        * Disable the :kconfig:option:`CONFIG_UART_CONSOLE` and :kconfig:option:`CONFIG_SERIAL` Kconfig options.

Reviewer: Is there a reason why these two got indented?

Author: Yes. I put everything in here under a toggle, so everything needs to be indented one level more.
